matlab/lib/+Ice/CFNetworkException.m

```diff
@@ -1,35 +0,0 @@
-% CFNetworkException   Summary of CFNetworkException
```

Comment:
I deleted a few exceptions that:
 - I plan to delete from C++ soon (if not already)
 - can't be thrown from the C++ client runtime